apps/expo-go/ios/Exponent/Versioned/Core/EXVersionUtils.mm

```diff
@@ -30,17 +16,10 @@ + (nonnull void *)versionedJsExecutorFactoryForBridge:(nonnull RCTBridge *)bridg
 {
   [bridge moduleForClass:[RCTUIManager class]];
 
-  EX_WEAKIFY(self);
-  const auto executor = [EXWeak_self, bridge](facebook::jsi::Runtime &runtime) {
-    if (!bridge) {
-      return;
-    }
-    EX_ENSURE_STRONGIFY(self);
-  };
   if ([jsEngine isEqualToString:@"hermes"]) {
-    return new facebook::react::HermesExecutorFactory(RCTJSIExecutorRuntimeInstaller(executor));
+    return new react::HermesExecutorFactory(react::RCTJSIExecutorRuntimeInstaller(nullptr));
```

Comment:
is it okay to pass nullptr here? it looks like we need some setup?
https://github.com/facebook/react-native/blob/c430083fa00338644390b74bdb48e68c8bbf648d/packages/react-native/Libraries/AppDelegate/RCTAppSetupUtils.mm#L144-L153